test-extension/background.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
```

Comment:
```suggestion
        <title>Background Page</title>
```

Reply:
This page is used for debugging, no need to change it cause it is not part of the library itself.